### ISSUE TITLE
Implement YOLO training dataset and batch loader

### ISSUE THREAD

**Issue by JYP-1317** [etri-ondev/ondevice-distance-estimation#17]
Dataset and DataLoader implementations are needed to validate and augment YOLO-format images and labels and serve them as training batches.
@HanQueue

**Comment by HanQueue** [comment]
Got it. I'll take the dataset and dataloader implementation.
